lib/modules/datasource/golang-version/__snapshots__/index.spec.ts.snap

```diff
@@ -3,7 +3,7 @@
 exports[`modules/datasource/golang-version/index getReleases parses real data 1`] = `
 {
   "homepage": "https://go.dev/",
-  "registryUrl": "https://raw.githubusercontent.com/golang/website/",
+  "registryUrl": "https://raw.githubusercontent.com/golang/website",
```

Comment:
Let's set the defaultRegistryUrls value to also be missing the trailing slash so that they are the same value